src/filesystem/File.js

```diff
@@ -31,7 +31,7 @@ define(function (require, exports, module) {
     var FileSystemEntry = require("filesystem/FileSystemEntry"),
         Content         = require("extensions/default/brackets-browser-livedev/nohost/src/content"),
         Path            = require("filesystem/impls/filer/BracketsFiler").Path,
-        defaultHTML     = require("text!filesystem/impls/lib/default.html");
+        defaultHTML     = require("text!extensions/default/brackets-browser-livedev/lib/default.html");
```

Comment:
Filed an [issue] (https://github.com/humphd/brackets/pull/170) in extension brackets-browser-livdev to do the opposite (i.e. remove the default.html file from there and use the one in bramble instead)